docs/elements/extensions.md

```diff
@@ -29,7 +29,7 @@ The `info.json` file is structurally similar to the element info file and may co
 }
 ```
 
-See the [reference for `infoElementExtension.json`](./schemas/infoElementExtension.md) for more information about what can be added to this file.
+See the [reference for `infoElementExtension.json`](../schemas/infoElementExtension.md) for more information about what can be added to this file.
```

Comment:
I moved around the `questions/runtime.md` and `elements/extensions.md`, `elements/dev.md` while I was here.